### PR TITLE
Add missing Drops

### PR DESCRIPTION
**Pull request by JulienItard**

### CODE REVIEW

**Comment by ZenCocoon** [comment]
can you please rebase

File: content/reference/objects/bathroom.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Bathroom
+
+Available attributes:
+
+* available_amenities <small>_(aliases as "amenities")_</small>
+* bath_count
+* count
```
Comment (ZenCocoon) [inline content/reference/objects/bathroom.md]: Where did you found this one?

File: content/reference/objects/sort.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Sort
+
+Available attributes:
+
+* sort_type
```
Comment (ZenCocoon) [inline content/reference/objects/sort.md]: `selected` is also present.

File: content/reference/objects/bathroom.md (hunk 1 above)
Comment (JulienItard) [inline content/reference/objects/bathroom.md]: Good question.

**Comment by ZenCocoon** [comment]
@JulienItard can you please squash this PR. (using FIX as comments from 2 latest commits is not needed)

**Pull request closed without merging**